src/root_files/_common/404.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Binary.com</title>
+    <meta charset="utf-8" />
+    <script type="text/javascript">
+    function getCookieItem(sKey) {
+        'use strict';
+        return decodeURIComponent(document.cookie.replace(new RegExp("(?:(?:^|.*;)\\s*" + encodeURIComponent(sKey).replace(/[\-\.\+\*]/g, "\\$&") + "\\s*\\=\\s*([^;]*).*$)|^.*$"), "$1")) || null;
+    }
+
+    var langs = {
+        'de': 'de|at|li',
+        'es': 'ar|bo|cl|co|cr|cu|do|ec|sv|gt|hn|mx|ni|pa|py|pr|es|uy|ve',
+        'fr': 'fr|ad|bj|bf|cf|cg|ga|gn|ml|mc|ne|sn|tg',
+        'it': 'it',
+        'ko': 'kr',
+        'pl': 'po',
+        'pt': 'br|mz|ao|pt|gw|pg|cv|st',
+        'ru': 'ru|ua|by|kz',
+        'th': 'th',
+        'vi': 'vn',
+        'zh_tw': 'tw|mo',
+        'zh_cn': 'cn',
+    };
+    var all_languages = ['en'].concat(Object.keys(langs));
+    function getLanguage(country) {
+        var conLang = 'en';
+        if (country) {
+            var keys = Object.keys(langs);
+            for (var i = keys.length; i--; ) {
+                var l = keys[i];
+                if ((new RegExp(langs[l], 'i')).test(country)) {
+                    return l;
+                }
+            }
+        }
+        return conLang;
+    }
+
+    var routeHandler = (function routeHandler() {
+        var location = window.location;
+
+        return {
+            isApp: function() {
+                return !location.href.includes('/app/');
```

Comment:
This path is only for `app_2`. So, if we add other sections in the future, this won't return a correct value. Preferably, change it to `isApp2()` to check for that section only, as well as the following `if..else`:
https://github.com/binary-com/binary-static/blob/793ab5161e11f5dceb68de6ac15df2a266116820/src/root_files/_common/404.html#L101-L107